inject/inject.vala

```diff
@@ -59,7 +61,25 @@ namespace Frida.Inject {
 			script_source = read_stdin ();
 		}
 
-		application = new Application (target_pid, target_name, script_path, script_source, enable_jit, enable_development);
+		var parameters = new Json.Node (Json.NodeType.OBJECT);
+		parameters.set_object (new Json.Object ()); 
+		if (parameters_str != null) {
+			if (parameters_str == "") {
+				printerr ("Parameters argument must be specified as JSON if present\n");
+				return 4;
+			}
```

Comment:
Why not let it fail below – or is the error message unintuitive?

Reply:
It seg faults.. so very unintuitive.

Reply:
🤣 Sounds like a bug in JSON-GLib. Anyway, let's work around it for now.